easybuild/easyblocks/n/numpy.py

```diff
@@ -66,7 +105,9 @@ def __init__(self, *args, **kwargs):
         self.sitecfg = None
         self.sitecfgfn = 'site.cfg'
         self.testinstall = True
-        self.testcmd = "cd .. && %(python)s -c 'import numpy; numpy.test(verbose=2)'"
+        # LDFLAGS should not be set when testing numpy/scipy, because it overwrites whatever numpy/scipy sets
+        # see http://projects.scipy.org/numpy/ticket/182
+        self.testcmd = "unset LDFLAGS && cd .. && %(python)s -c 'import numpy; numpy.test(verbose=2)'"
```

Comment:
@Flamefire To convert the Boolean return value to an exit code, would the suggestion here be something like:
```
self.testcmd = "unset LDFLAGS && cd .. && %(python)s -c 'import numpy; import sys; sys.exit(!int(numpy.test(verbose=2))'"
```

Reply:
Yes, that would likely work. To keep the changes and the work done here I'd suggest to extract that change (likely the only one required) to a new PR and test it with a couple numpys

Reply:
Agree, please keep the `unset LDFLAGS` and associated comment (which would have solved the issue you ran into)

Reply:
Done: #2238